_layouts/role.html

```diff
@@ -9,21 +9,45 @@
 {% assign personas = site.personas | where: "roles", page.title %}
 {% assign projects = site.projects | where: "roles", page.title %}
 {% assign posts = site.posts | where: "roles", page.title %}
+{% assign books = site.data.books.docs | where: "roles", page.title %}
+{% assign tools = site.data.tools.docs | where: "roles", page.title %}
 {% include breadcrumbs.html %}
 <div class="guide-main">
     <div class="">
         <div class="grid-container">
             <div class="grid-row grid-gap flex-align-start">
+                <!--
                 {% assign sidenav = site.roles.navigation[page.sidenav] | default: page.sidenav %}
                 {% if sidenav %}
                 <div class="usa-layout-docs__sidenav desktop:grid-col-3{% if page.sticky_sidenav %} usa-sticky-sidenav{% endif %}">
                     {% include sidenav-roles.html links=sidenav %}
                 </div>
                 {% endif %}
+            -->
                 <div class="usa-layout-docs usa-layout-docs__main usa-prose{% if sidenav %} desktop:grid-col-9{% endif %}">
+                    <!--
```

Comment:
Removing this is causing an accessibility error

```
Errors in http://localhost:4000/roles/executive:

 • Page must contain a level-one heading
   (https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one?application=axeAPI)

   (html)

   <html lang="en-US" class="fa-events-icons-ready"><head> <meta
   name="viewport...</html>
```

Reply:
@dmundra ah, I just did it for one page in the front matter and didn't think it would impact all of them. Should work now.